src/app/booking/ActivitiesCalendar.js

```diff
@@ -230,6 +233,64 @@ export function ActivityCard({ activity, targetDate }) {
   );
 }
 
+export function EmblaContainer({ emblaSx: emblaSxOuter, children }) {
```

Comment:
<img width="1066" alt="image" src="https://github.com/digital-brother/ooscca-fe/assets/43747573/25cbe333-8673-4363-a999-eb1e09dcf278">
Invalid alignmenet for mobile